docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -162,6 +164,9 @@
     cj_strength=0.5,
 )
 
+# Use SimCLR augmentations with larger image size for SimMIM
+simmim_transform = SimCLRTransform(input_size=224)
+
```

Comment:
Added this transform, otherwise the benchmarks crash if the input size is not set to 224 which is not the default.